app/views/users/index.html.erb

```diff
@@ -39,12 +23,12 @@
             <td><%= user.address %></td>
             <td>
               <% user.organizations.each do |organization| %>
-                <p><%= organization.name %></p>
+                <span><%= organization.name %></span>
```

Comment:
Ah, this was only a styling change.  The P tags had too much margin that was causing the row heights to vary from row to row.  My only concern was uniform row height.  Is a P tag the best way to ensure vertical alignment across rows?